p2p/state.py

```diff
@@ -109,6 +130,21 @@ def msg_queue_maxsize(self) -> int:
             peer = cast(ETHPeer, peer)
             asyncio.ensure_future(self._handle_msg(peer, cmd, msg))
 
+    async def _process_nodes(self, nodes: Iterable[Tuple[Hash32, bytes]]) -> None:
+        for idx, (node_key, node) in enumerate(nodes):
+            self._total_processed_nodes += 1
+            try:
+                self.scheduler.process([(node_key, node)])
+            except SyncRequestAlreadyProcessed:
+                # This means we received a node more than once, which can happen when we
+                # retry after a timeout.
+                pass
+            if idx % 10 == 0:
+                # XXX: This is a quick workaround for
+                # https://github.com/ethereum/py-evm/issues/1074, which will be replaced soon
+                # with a proper fix.
+                await self.sleep(0)
```

Comment:
should this be `self.sleep(0)`?

Reply:
hmm, what do you mean? that's what it is

Reply:
No clue, I must have looked at this and read it as `asyncio.sleep`.  Disregard.